xbmc/FileItem.cpp

```diff
@@ -1270,7 +1270,7 @@ bool CFileItem::IsAndroidApp() const
 
 bool CFileItem::IsStack() const
 {
-  return URIUtils::IsStack(m_strPath);
+  return URIUtils::IsStack(m_strDynPath);
```

Comment:
Please use `GetDynPath()` like in my patch since it defers to m_strPath if m_strDynPath is not yet populated.

Reply:
Done